src/Akka.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
+VisualStudioVersion = 14.0.25420.1
```

Comment:
Not sure if it was this change that caused this, but the mono build failed to execute as a result of some of the changes in this PR.

```
[18:11:15]	[docker]   1) Building /checkout/src/Akka.sln failed with exitcode 1.
[18:11:15]	[docker]   2) CS1061: Transport/AkkaIO/AddressConverters.cs(24,86): Type `System.Net.IPAddress' does not contain a definition for `MapToIPv4' and no extension method `MapToIPv4' of type `System.Net.IPAddress' could be found. Are you missing an assembly reference?
[18:11:15]	[docker]   3) CS2001: CSC(0,0): Source file `Dsl/Flow.cs' could not be found
```